Signal/src/util/CenterStageUtil.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  Copyright (c) 2020 Open Whisper Systems. All rights reserved.
+//
+
+import Foundation
+
+class CenterStageUtil {
```

Comment:
I'm not really understanding why this class uses KVO, or why it's a class at all. It should probably be a `struct` (or even an `enum` since it's never initialized) and it could have something like

```swift
static var centerStageEnabled: Bool {
didSet {  do what you need to do }
}

Reply:
There are two ways for user to switch on/off center stage: via control center, or via in app UI. While the user is on a call, potentially the user could access the control center, and press the Center stage button. The KVO listening is to keep the control center and app UI in sync.

Reply:
For class vs struct, I was under the impression that structs are meant for data structures. The class here is merely a means to group functionality, in a way that clearly communicates to the point of call what the method relates to (Center stage).